traveltimepy/mapper.py

```diff
@@ -118,6 +121,92 @@ def create_time_filter(
         raise ApiError("arrival_time or departure_time should be specified")
 
 
+def create_time_map_fast(
+    coordinates: List[Coordinates],
+    transportation: time_map_fast.Transportation,
+    travel_time: int = 3600,
+    one_to_many: bool = False,
+    snapping: Optional[Snapping] = None,
+) -> TimeMapFastRequest:
+    if one_to_many:
+        return TimeMapFastRequest(
+            arrival_searches=time_map_fast.ArrivalSearches(
+                one_to_many=[
+                    time_map_fast.Search(
+                        id=f"Search {ind}",
+                        coords=cur_coordinates,
+                        transportation=transportation,
+                        travel_time=travel_time,
+                        arrival_time_period="weekday_morning",  # TODO: make customizable with enum / literal
```

Comment:
time filter fast has this hardcoded, so keeping it like this for now. Will make it customizable (a literal to avoid breaking changes) in the future

Reply:
not necessarily a literal, but it has to be optional, with default value of "weekday_morning"